src/xtgeo/xyz/_xyz_roxapi.py

```diff
@@ -80,7 +80,7 @@ def _check_input_and_version_requirement(
                 f"Invalid {category=}. Valid entries are {VALID_WELL_PICK_TYPES}"
             )
         if is_polygons:
-            raise ValueError(f"Polygons does not support {stype=}.")
+            raise ValueError(f"Polygons does not support {stype=}.")  # stype.value?
```

Comment:
This will give the intended behaviour 🙂 
```suggestion
            raise ValueError(f"Polygons does not support stype={stype.value}.")  
```